app/models/user.rb

```diff
@@ -386,6 +386,13 @@ def all_blocking
     UserBlock.where(blocker_id: id)
   end
 
+  def cached_all_blocking_user_ids
```

Comment:
Given we are already caching the [user_data](https://github.com/thepracticaldev/dev.to/pull/4678/files#diff-02a7d87efea39bec38ecc2737b8eba6cR34) and that is the only placed this is called what is the thought behind caching this again here?

Reply:
Hmm that's a good question. Maybe @benhalpern has thoughts?

Reply:
My thoughts are not to use the additional cache to start and then add it later if needed. Having a second cache here adds another layer of complexity bc now we have to worry about clearing this cache when updates are run and unless that added complexity is really giving us a big performance boost I dont think it's worth it.